src/test/java/uk/nhs/prm/deduction/e2e/TestConfiguration.java

```diff
@@ -181,6 +181,10 @@ public String getEhrRepoUrl() {
         return String.format("https://ehr-repo.%s.patient-deductions.nhs.uk/", getEnvSuffix());
     }
 
+    public String getEhrOutUrl() {
+        return String.format("https://ehr-out-service.%s.patient-deductions.nhs.uk/", getEnvSuffix());
```

Comment:
Could this not just be a constant?

```suggestion
public static String EHR_OUT_URL = String.format("https://ehr-out-service.%s.patient-deductions.nhs.uk/", getEnvSuffix());
```

Reply:
Additionally, reflect throughout the URLs :)

Reply:
Thank you!
I have addressed this by changing the urls into static constants and updating the references in the new tests that we written.

There are some references to those getXXXUrl methods in other test suites which is not in the scope of this ticket, so I added a TODO comment to address those after we finish moving all packages to main.